splash.h

```diff
@@ -0,0 +1,121 @@
+  0b00000000,0b00000000,0b00001111,0b10001111,0b10000000,0b00000000,0b00000000,0b00000011,0b11100000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000011,0b00001111,0b00000000,0b00000000,0b00000000,0b00000001,0b10000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00011111,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00111110,0b00000000,0b11111111,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b01111100,0b00000011,0b11111111,0b11000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00111000,0b00011111,0b11111111,0b11100000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00011000,0b01111111,0b11111111,0b11111100,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b11111111,0b11111111,0b11111110,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000011,0b11111111,0b11111111,0b11111111,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000011,0b11111111,0b11111111,0b11111111,0b10000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000001,0b11111111,0b11111111,0b11111111,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b11111111,0b11111111,0b11111100,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b01111111,0b11111111,0b11111100,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00111111,0b11111111,0b11110000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00011111,0b11111111,0b11110000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00001111,0b11111111,0b11100000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000111,0b11111111,0b11000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000011,0b11111111,0b10000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000001,0b11111110,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b11111110,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b01111100,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b00110000,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+  0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,0b00110000,0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+};
+
+#define splash2_width  115
+#define splash2_height 32
+
+const uint8_t PROGMEM splash2_data[] = {
```

Comment:
Is this still the Adafruit Logo from the original library?

If so please replace it with a senseBox logo.

Reply:
splash2_data is still the adafruit logo

Reply:
Then please add a senseBox logo for `splash2_data` too.